### ISSUE TITLE
settleAsync() and allAsync() only handle GET requests

### ISSUE THREAD

**Issue by blowback** [lucasfeliciano/restling#3]
I tried making a request object with `options: { method: 'POST' }` but it looks like these methods only support GET if I'm reading `wrapToArray()` and `wrapToObject()` correctly.

It would be useful to be able to pass the method in the options object.

**Comment by lucasfeliciano** [comment]
Hey @blowback,

First of all thank you to have some time to open a issue.

Yeah, you are right, first I implemented it just to use as GET for my necessity.
Since you brought this subject to the table I'll implement it and this feature should be in the next release.

**Comment by lucasfeliciano** [comment]
Spec:

To change that behaviour we need refactor these  two functions below to verify the if the user passed the object with the key `method` and after that validation call the correspondent function to each one.

```
function wrapToArray(myRequests) {
  return _.map(myRequests, function(myRequest){
    return exports.get(myRequest.url, myRequest.options);
  });
}

function wrapToObject(myRequests){
  return _.mapValues(myRequests, function(myRequest){
    return exports.get(myRequest.url, myRequest.options);
  })
}
```

**Comment by lucasfeliciano** [comment]
Hey @blowback, just released version 0.8.0 which have this feature implemented. ( 12a1f705587728cbbe1b4c3d073693754e9d9e15 )
Give me feedback if everything it is ok for you.
